doc/_admin-guide/120_Parser/018_websense_parser.md

```diff
@@ -19,11 +19,11 @@ For example:
 
 {% include doc/admin-guide/parser-support.md %}
 
-The {{ site.product.short_name }} application sets the ${PROGRAM} field to Websense.
+The {{ site.product.short_name }} application sets the PROGRAM field to Websense.
 
 By default, the websense-specific fields are extracted into name-value
-pairs prefixed with .websense. For example, the product\_version in the
-previous message becomes ${.websense.product\_version}. You can change
+pairs prefixed with .websense. For example, the product_version in the
```

Comment:
in this sentence you use both the ${macro_name} and the macro_name syntax, this is somewhat confusing and might suggest additional parsing, but both are macros

Reply:
Will be fixed in #71